test/images/agnhost/netexec/netexec.go

```diff
@@ -37,19 +37,23 @@ import (
 	"github.com/spf13/cobra"
 
 	utilnet "k8s.io/apimachinery/pkg/util/net"
+	"k8s.io/gengo/examples/set-gen/sets"
```

Comment:
```suggestion
	"k8s.io/apimachinery/pkg/util/sets"
```

Reply:
I should never trust vscode when I choose a package

Reply:
Done